nodes/cma/nodes-cma-autoscaling-custom.adoc

```diff
@@ -52,3 +52,21 @@ image::564_OpenShift_Custom_Metrics_Autoscaler_0224.png[Custom metrics autoscale
 6. As a it operates, a workload can affect the scaling metrics. For example, if a workload is scaled up to handle work in a Kafka queue, the queue size decreases after the workload processes all the work. As a result, the workload is scaled down.
 
 7. If the metrics are in a range specified by the `minReplicaCount` value, the custom metrics autoscaler controller disables all scaling, and leaves the replica count at a fixed level. If the metrics exceed that range, the custom metrics autoscaler controller enables scaling and allows the HPA to scale the workload. While scaling is disabled, the HPA does not take any action.
+
+[id="nodes-cma-autoscaling-custom-ca_{context}"]
+== Custom CA certificates for the Custom Metrics Autoscaler
+
+By default, the Custom Metrics Autoscaler Operator uses automatically-generated service CA certificate to connect to on-cluster services. 
+
+If you want to use off-cluster services that require custom CA certificates, you can add the required certificates to a config map. Then, add the config map to the `KedaController` custom resource as described in "Installing the custom metrics autoscaler". The Operator loads those certificates on start-up, which are registered as trusted by the Operator. 
```

Comment:
* could directly xref and remove the _Additional resources_ section since this is in an assembly
* suggest tweaking the last sentence, the reference is a bit odd to parse
```suggestion
If you want to use off-cluster services that require custom CA certificates, you can add the required certificates to a config map. Then, add the config map to the `KedaController` custom resource as described in xref:../../nodes/cma/nodes-cma-autoscaling-custom-install.adoc#nodes-cma-autoscaling-custom-install[Installing the custom metrics autoscaler]. The Operator loads those certificates on start-up and registers them as trusted by the Operator. 
```